packages/teleterm/src/ui/TopBar/Identity/IdentitySelector/IdentitySelector.tsx

```diff
@@ -17,12 +18,18 @@ export const IdentitySelector = forwardRef<
   HTMLButtonElement,
   IdentitySelectorProps
 >((props, ref) => {
+  const { getLabelWithShortcut } = useKeyboardShortcutFormatters();
   const isSelected = props.userName && props.clusterName;
   const text = isSelected && `${props.userName}@${props.clusterName}`;
   const Icon = props.isOpened ? SortAsc : SortDesc;
 
   return (
-    <Container isOpened={props.isOpened} ref={ref} onClick={props.onClick}>
+    <Container
+      isOpened={props.isOpened}
+      ref={ref}
+      onClick={props.onClick}
+      title={getLabelWithShortcut('Open Profile', 'toggle-identity')}
+    >
```

Comment:
Hmm, this makes it hard to discover the actual shortcut, because the tooltip is different when you hover on the avatar or the arrow and the profile name itself.

Could we make it so that there's only one tooltip for this Container? Something along the lines of `Open Profile <profile name> <shortcut>` I suppose?

Reply:
You are right here, having two tooltips for the Container makes one of them hard to access. I discovered that I can use line break in the `title` attribute, so the final tooltip will look like this:
```
<profile name>
Open Profiles <shortcut>
```